app/components/molecules/promotion-card/promotion-card.component.tsx

```diff
@@ -0,0 +1,32 @@
+import {FC} from 'react';
+
+import {BadgeContainer, Img, Title, Card} from './promotion-card.styles';
+import {Badge} from '../../atoms/badge';
+
+interface PromotionCardProps {
+  image: string;
+  isRecommended?: boolean;
```

Comment:
`recommended` without the `is` would be a pain? 🤔

Reply:
I think the `is` prefix is a good indicator that the prop expects a boolean value without having to look into the component's/props interface's definition.